src/events/swipe-handler.js

```diff
@@ -59,6 +59,12 @@ export default (
         }, { passive: true });
 
         $touchTarget.addEventListener('touchend', (event: TouchEvent) => {
+            if (!distX && !distY) {
+                // this is to fix the bug mentioned in this:
```

Comment:
Remove this comment please, since JIRA is a private instance and this lib is open source :-D